src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -230,6 +230,12 @@ ov::OutputVector reduce_l2(const Node& node) {
     return {make_ov_reduction_op<v4::ReduceL2>(node, node.get_ov_inputs().at(0), supported_types_v2)};
 }
 
+ov::OutputVector reduce_log_sum_exp(const ov::frontend::onnx::Node& node) {
+    const auto exp_node = std::make_shared<v0::Exp>(node.get_ov_inputs().at(0));
+    const ov::Output<ov::Node> sum_node = make_ov_reduction_op<v1::ReduceSum>(node, exp_node, supported_types_v2);
+    return {std::make_shared<v0::Log>(sum_node)};
```

Comment:
In some cases, it may lead to overflow issue. I think you need log-sum-exp trick to avoid it for stability: https://gregorygundersen.com/blog/2020/02/09/log-sum-exp/

Reply:
I guess it is better to fix them both, but as a separate pr. Here I'm just allow to have "axes" as an input as it is required in opset-18